### PR TITLE
hpcc4j-312 Manually fix javadoc errors and warnings

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexisrisk.com>

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@rpastrana please review

File: DataAccess/src/main/java/org/hpccsystems/spark/HpccRDD.java [hunk 1]
```diff
@@ -189,8 +190,8 @@ public HpccRDD(SparkContext sc, DataPartition[] dataParts, FieldDef originalRD,
     /**
      * Transform to mllib.linalg.Vectors for ML Lib machine learning.
      * @param dimNames the field names for the dimensions
-     * @throws IllegalArgumentException
-     * @return
+     * @throws IllegalArgumentException illegal argument exception
+     * @return jrdd.map row object
```
Comment (rpastrana) [inline DataAccess/src/main/java/org/hpccsystems/spark/HpccRDD.java:194]: RDD Vector object?

**Review by rpastrana — COMMENTED** [review]
@Michael-Gardner looks good, just one comment

**Comment by richardkchapman** [comment]
@Michael-Gardner This seems to have stalled?

File: DataAccess/src/main/java/org/hpccsystems/spark/HpccRDD.java (hunk 1 above)
Comment (Michael-Gardner) [reply DataAccess/src/main/java/org/hpccsystems/spark/HpccRDD.java]: Fixed to vector object

**Review by rpastrana — APPROVED** [review]
@Michael-Gardner approved

**Pull request merged**